www/plugins/examples/busy.js

```diff
@@ -14,6 +14,8 @@ module.exports = function (widgetName) {
     <>
       <${widgetName} busy />
       <br/>
+      <${widgetName} busy hideCaret />
```

Comment:
I believe this example is used by multiple components, not all of which have "hideCaret" as a prop. We'd want to make a combobox specific one, or probably better make this part of a `hideCaret` example instead.

Reply:
revert this?

Reply:
Done.